mkdocs.yml

```
@@ -92,4 +92,5 @@ nav:
         - wiki/Sudo.md
         - wiki/Terminals.md
         - wiki/JIT-Debugging.md
-    - 'contact.md'
\ No newline at end of file
+    - 'contact.md'
+    - 'codeofconduct.md'
```

A new line at the end of file :)
```suggestion
    - 'codeofconduct.md'

```